### ISSUE TITLE
[JOSS Review] Paper comments

### ISSUE THREAD

**Issue by CFGrote** [zhanghe9704/tpsa#15]
JOSS review https://github.com/openjournals/joss-reviews/issues/4818

Please take these as suggestions to improve clarity and readability of the paper

## Abstract
- I would remove the part ", based on which more sophisticated methods ..., etc." and put these references into the Background section (citations in abstracts should be self-contained, in any case). 
- Instead, add a short statement of need, along the lines of "a reusable implementation of tpsa/da as a modern c++ library or other high level programming language like python is currenty missing or incomplete"
## Background
The Background section is excellent, well written, concise and accessible to any non-expert reader. Well done!
- line 34 (page 2) has a typo, should read "limited number of iterations".
## Statement of need
- line 44: avoid "etc" to conclude citations. It does not add any information to the reader.
- In general, there are a number of typos and language issues in this section. If possible, please consult a native speaker. AI tools are your friend, too ;)
## Features
- line 63: please add a reference or URL to L. Yang's code. Up to which version was it part of MAD-X?
## Verification:
- If possible, can you quantify the (average) discrepancy between cosy-infinity and your tpsa/da? e.g. RMS or other distance metric? This would make your assessment more comparable to other tpsa/da implementations.
- Are there any potential/hypothetical applications, where errors as small as 1e-15 or 1e-16 could become important?

**Comment by zhanghe9704** [comment]
**Abstract** 
- removed the references and revised the abstract as suggested. 

**Background**
- corrected the typo

**Statement of need**
- checked the grammar and revised it. Hope all errors are caught. 

**Features**
- Added the reference to Yang's code. I checked the MAD-X repo. Found Yang's code is still used in the latest release, not as an individual module as before but inside the PTC module. Changed the statement. 

**Verification** 
- Actually, in most cases the two programs agree to the machine precision because all the fundamental operators are defined and carried out in the same way. The differences only happen when different numerical methods are used, such as a special function may be approximated using different series. 
-  Considering A TPSA vector as a Taylor expansion of a function, there is a truncation error in theory. If very high precision is required, I think one should/could use the TM model in COSY Infinity, which calculates a TPSA vector together with a rigorous error band. But this is outside the scope of this TPSA lib. 
- Revised this part to include the above notes.
